utils/bash-completion/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2024 CZ.NIC, z.s.p.o.
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bash-completion
+PKG_VERSION:=2.14.0
+PKG_RELEASE:=0
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://www.github.com/scop/bash-completion/releases/download/${PKG_VERSION}
+PKG_HASH:=5c7494f968280832d6adb5aa19f745a56f1a79df311e59338c5efa6f7285e168
+
+PKG_MAINTAINER:=Michal Hrusecky <michal.hrusecky@turris.com>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/bash-completion
+  SECTION:=utils
+  CATEGORY:=Utilities
+  SUBMENU:=Shells
+  TITLE:=Collection of command line command completions for Bash shell
+  DEPENDS:=+bash
```

Comment:
The upstream package README states:

```
Many of the completion functions assume GNU versions of the various text utilities that they call (e.g. grep, sed, and awk). Your mileage may vary.
```

Yet your makefile has no dependencies on neither built-in nor GNU versions of those utilities.

Reply:
Well, because the dependencies will vary. In theory the best approach (and really tedious one) would be to split it per utility, but then it would be really hard to do in the first place and especially maintain in the long run... Will think about it.